sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-BCAG.e2e-spec.ts

```diff
@@ -0,0 +1,75 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+import { CredentialType, InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG.`, () => {
+  it(
+    "Should determine BCAG as eligible when total assessed need is greater than 0 " +
```

Comment:
"... when total assessed is greater than..." ?

Reply:
Actually the variable is called `calculatedDataTotalAssessedNeed`
![image](https://github.com/bcgov/SIMS/assets/77353155/d3c07119-e8aa-4c4b-922a-f6a4f3ad565b)